magento/pwa-studio
# [feature]: allow blob: types for style-src

Fix issue https://github.com/magento/pwa-studio/issues/2979



## Description

Added 'blob' keyword to veniaSecurityHeaders default template, enabling the load of blob stylesheet contents under CSP constraints.

## Related Issue





Closes #2979 

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. yarn build
2. yarn start
3. verify that blob stylesheet is loading without browser csp errors

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
